searchlib/src/vespa/searchlib/query/streaming/dot_product_term.cpp

```diff
@@ -51,8 +51,9 @@ DotProductTerm::unpack_scores(Scores& scores, std::optional<double> score_thresh
 }
 
 void
-DotProductTerm::unpack_match_data(uint32_t docid, const ITermData& td, MatchData& match_data)
+DotProductTerm::unpack_match_data(uint32_t docid, const ITermData& td, MatchData& match_data, const fef::IIndexEnvironment& index_env)
 {
+    (void) index_env;
```

Comment:
When the type is specific you do not need a name for the parameter, and then you do not need the explicit (void).